contracts/VRTVault/VRTVault.sol

```diff
@@ -117,6 +117,7 @@ contract VRTVault is VRTVaultStorage, ReentrancyGuard {
 
         user.accrualStartBlockNumber = getBlockNumber();
         emit Deposit(userAddress, depositAmount);
+        vrt.safeIncreaseAllowance(address(this), depositAmount);
```

Comment:
Why do we need it?

Reply:
@kkirka @RoseRompkxm 

https://app.clickup.com/t/2cy2qpq

the suggestion passed was 

```
During the code review, It has been observed that allowance checks are redundant. If the VRT does not have enough allowance for the transfer, the transaction fails. 
```

```
Consider reviewing the workflow, before transfer the msg.sender should approve the address via defined amount. It is recommended to use safeIncrease and safeDecreaseAllowance in the related function. 
```

this was passed for VRTConversion (where user approves spending of VRT followed by `convert`)
the same applies for VRTVault (deposit VRT)


do we need it?
if yes, is it to:
1. increaseAllowance
2. transfer
3. followed by decreaseAllowance?

Reply:
@kkirka Removed the function call changes